### PR TITLE
changed detection technique to dlib

### PR DESCRIPTION
**Pull request by SonaliSinha20**
changed face detection technique to dlib

### CODE REVIEW

File: faster_facial_landmarks.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+TEXT_SIZE=0.7
+TEXT_WIDTH=2
+REC_FACE_BLUE=255
+REC_FACE_GREEN=0
+REC_FACE_RED=0
+REC_FACE_WIDTH=2
+REC_CENTER_X=350
+REC_CENTER_Y=250
+# vs = VideoStream(usePiCamera=True).start() # Raspberry Pi
+time.sleep(2.0)
```
Comment (apurva1112) [inline faster_facial_landmarks.py:53]: declare the constant 2.0 and use

File: faster_facial_landmarks.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+    gray = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+ 
+    # detect faces in the grayscale frame
+    rects = detector(gray, 0)
+
+    # check to see if a face was detected, and if so, draw the total
+    # number of faces on the frame
+    if len(rects) > 0:
+        text = "{} face(s) found".format(len(rects))
+        cv2.putText(frame, text, (10, 20), cv2.FONT_HERSHEY_SIMPLEX,0.5, (0, 0, 255), 2)
```
Comment (apurva1112) [inline faster_facial_landmarks.py]: declare the constants

File: faster_facial_landmarks.py [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+    # loop over the face detections
+    for rect in rects:
+    # compute the bounding box of the face and draw it on the
+        # frame
+        (bX, bY, bW, bH) = face_utils.rect_to_bb(rect)
+        cv2.rectangle(frame, (bX, bY), (bX + bW, bY + bH),(0, 255, 0), 1)
```
Comment (apurva1112) [inline faster_facial_landmarks.py]: declare the constants

File: faster_facial_landmarks.py [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+            # determine the facial landmarks for the face region, then
+            # convert the facial landmark (x, y)-coordinates to a NumPy
+            # array
+        shape = predictor(gray, rect)
+        shape = face_utils.shape_to_np(shape)
+ 
+            # loop over the (x, y)-coordinates for the facial landmarks
+            # and draw each of them
+        for (i, (x, y)) in enumerate(shape):
+            cv2.circle(frame, (x, y), 1, (0, 0, 255), -1)
```
Comment (apurva1112) [inline faster_facial_landmarks.py]: declare the constants

File: faster_facial_landmarks.py [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+            cv2.putText(frame, str(i + 1), (x - 10, y - 10),
```
Comment (apurva1112) [inline faster_facial_landmarks.py:106]: declare the constants

File: faster_facial_landmarks.py [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+            cv2.FONT_HERSHEY_SIMPLEX, 0.35, (0, 0, 255), 1)
```
Comment (apurva1112) [inline faster_facial_landmarks.py]: declare the constants

File: faster_facial_landmarks.py (hunk 1 above)
Comment (AMM12) [reply faster_facial_landmarks.py:53]: ok sir

**Pull request merged**